railties/test/commands/statistics_test.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+
+class Rails::Command::DevTest < ActiveSupport::TestCase
+  setup :build_app
+  teardown :teardown_app
+
+  test "`bin/rails stats` handles non-existing directories added by third parties" do
+    Dir.chdir(app_path) do
+      app_file("lib/tasks/custom.rake", <<~CODE
+        task stats: "custom:statsetup"
+        namespace :custom do
+          task statsetup: :environment do
+            require "rails/code_statistics"
+            ::STATS_DIRECTORIES << ["app/components"]
```

Comment:
In case Rails ever adds a `app/components` directory maybe we should use a different name.
What do you think about:
```suggestion
            ::STATS_DIRECTORIES << ["app/custom"]
```
Or maybe it would be clearer with:
```suggestion
            ::STATS_DIRECTORIES << ["app/non_existing"]
```

Reply:
Good point. I changed it.